### ISSUE TITLE
[Features] Alert the editor when a request as been treated and create a dashboard to check if everything is ok.

### ISSUE THREAD

**Issue by tunisiano187** [tunisiano187/WME-send-to-slack#19]
The editors who are asking for an unlock or lock don't receive an answer when it's done

**Describe the solution you'd like**
Create a Spreadsheet that will store the last request of a segment and the state of the request (or the last known lock level)

**Describe alternatives you've considered**
Read this to see how to work with spreadsheet : https://greasyfork.org/fr/scripts/3776-wme-check-road-name

**Comment by tunisiano187** [comment]
Still interesting, but need more time

Le jeu. 16 juil. 2020 à 14:52, stale[bot] <notifications@github.com> a
écrit :

**Comment by tunisiano187** [comment]
Yes

Le ven. 31 juil. 2020 à 18:13, stale[bot] <notifications@github.com> a
écrit :

**Comment by ordonezgs** [comment]
Sharing an idea...

> After WMESTS permalink opened and after save triggered then edit  message as solved and add a message emoji to the message
In order to check for requests handled 

Not sure about a dashboard

**Comment by tunisiano187** [comment]
indeed, but it can be in a spreadsheet, that can be used as dashboard...
The problem is the write acces to the file...
